From: Stefano Rivera <stefanor@debian.org>
Date: Sun, 28 Mar 2021 21:44:55 -0700
Subject: Skip test_find_local which requires Internet access

test_find_local() calls pip (via pep517) to install setuptools in a new
environment. We do not permit Internet access during package builds.

Forwarded: not-needed
---
 tests/test_integration.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tests/test_integration.py b/tests/test_integration.py
index 00e9021..3430c45 100644
--- a/tests/test_integration.py
+++ b/tests/test_integration.py
@@ -45,6 +45,7 @@ class FinderTests(fixtures.Fixtures, unittest.TestCase):
 
 
 class LocalProjectTests(fixtures.LocalPackage, unittest.TestCase):
+    @unittest.skip("Requires Internet access for pip")
     def test_find_local(self):
         dist = Distribution._local()
         assert dist.metadata['Name'] == 'local-pkg'
